demo/full/styles/style.css

```diff
@@ -263,7 +260,7 @@ body {
 .control-button {
   width: 2em;
   border: none;
-  font-family: "icons";
+  font-family: "icons", sans-serif;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected missing generic font family ([css:S4649](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4649))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=264&issues=AWTMwmjYi6fIhhPMqNRW&open=AWTMwmjYi6fIhhPMqNRW)

[ISSUE_KEY]: AWTMwmjYi6fIhhPMqNRW